packages/react-composites/src/composites/CallWithChatComposite/ModalLocalAndRemotePIP.tsx

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import {
+  concatStyleSets,
+  ContextualMenu,
+  IDragOptions,
+  IModalStyleProps,
+  IModalStyles,
+  IStackStyles,
+  IStyleFunctionOrObject,
+  Modal,
+  Stack
+} from '@fluentui/react';
+import React, { useMemo } from 'react';
+import { CallAdapter } from '../CallComposite';
+import { CallAdapterProvider } from '../CallComposite/adapter/CallAdapterProvider';
+import { LocalAndRemotePIP } from '../CallComposite/components/LocalAndRemotePIP';
+import { useHandlers } from '../CallComposite/hooks/useHandlers';
+import { useSelector } from '../CallComposite/hooks/useSelector';
+import { localAndRemotePIPSelector } from '../CallComposite/selectors/localAndRemotePIPSelector';
+
+/**
+ * Drag options for Modal in {@link ModalLocalAndRemotePIP} component
+ */
+const DRAG_OPTIONS: IDragOptions = {
+  moveMenuItemText: 'Move',
+  closeMenuItemText: 'Close',
+  menu: ContextualMenu,
+  keepInBounds: true
+};
+
+/**
+ * Styles for {@link ModalLocalAndRemotePIP} component
+ */
+export type ModalLocalAndRemotePIPStyles = { modal?: Partial<IModalStyles> };
+
+const _ModalLocalAndRemotePIP = (props: {
+  hidden: boolean;
+  modalLayerHostId: string;
+  styles?: ModalLocalAndRemotePIPStyles;
+}): JSX.Element => {
+  const rootStyles = props.hidden ? hiddenStyle : PIPContainerStyle;
+  const pictureInPictureProps = useSelector(localAndRemotePIPSelector);
+  const pictureInPictureHandlers = useHandlers(LocalAndRemotePIP);
+  const localAndRemotePIP = useMemo(
+    () => <LocalAndRemotePIP {...pictureInPictureProps} {...pictureInPictureHandlers} />,
+    [pictureInPictureProps, pictureInPictureHandlers]
+  );
+
+  const modalStylesThemed = concatStyleSets(modalStyle, props.styles?.modal);
+
+  return (
+    <Stack styles={rootStyles}>
+      <Modal
```

Comment:
Maybe think about using ModalClone?

Reply:
Will do that in a separate PR because I would need to export `ModalClone` from 'react-composites'